src/identity/index.ts

```diff
@@ -1,10 +1,13 @@
-import { Transaction } from "@scure/btc-signer";
+import { Transaction } from "@scure/btc-signer/transaction.js";
 import { SignerSession } from "../tree/signingSession";
 
 export interface Identity {
+    signerSession(): SignerSession;
+    xOnlyPublicKey(): Promise<Uint8Array>;
+    compressedPublicKey(): Promise<Uint8Array>;
+    signMessage(message: string): Promise<Uint8Array>;
```

Comment:
```suggestion
    signMessage(message: Uint8Array): Promise<Uint8Array>;
```
so we don't have to handle encoding ?

Reply:
Nack.
One of the usages of this is to sign an address (string) to implement magic hint with Boltz swaps.
Don't see why we should add an extra burden to the user to encode the address into bytes to sign it.